### ISSUE TITLE
U86i without U86

### ISSUE THREAD

**Issue by pasqLisena** [DOREMUS-ANR/doremus-ontology#32]
Is intentional  this `U86i_was_creation_mode_of` without a `U86_xxxxxxxxxxx`?
see [line 1850](https://github.com/DOREMUS-ANR/doremus-ontology/blob/master/doremus.ttl#L1850) of the ontology

**Comment by rtroncy** [comment]
Do you mean that you found not normal to have only the inverse property defined (U86i) in the ontology and not the reciprocal property (U86)?

**Comment by pasqLisena** [comment]
> Do you mean that you found not normal to have only the inverse property defined (U86i) in the ontology and not the reciprocal property (U86)?

Yes

**Comment by pierrechoffe** [comment]
U86 used to be "foresees creation or performance mode", and applied both to Expressions and Events. This was wrong since an Event in CRM can only be in the past and therefore the property has to be in the past tense. The prior U86 also had U86i, so this is definitely a bug that happened when modifying U86.

**Comment by pasqLisena** [comment]
_ foresees creation or performance mode_ is [U90](https://github.com/DOREMUS-ANR/doremus-ontology/blob/master/doremus.ttl#L1917).

We use it on [F25 Performance Plan](http://data.doremus.org/sparql?default-graph-uri=&query=select+*+where+%7B+%3Fs+a+%3Fclass%3B+mus%3AU90_foresees_creation_or_performance_mode+%3Fo+%7D&should-sponge=&format=text%2Fhtml&timeout=0&debug=on). Is it correct?

**Comment by pierrechoffe** [comment]
From what I understand, the property `foresees creation or performance mode` would be used either on foreseen activities or on F25 or even on F22.
In the case of F22, it is supposed to describe the **composer's** intentions regarding certain performance modes.
In the case of foreseen performances, it is supposed to describe the **performers'** intentions when they are declared in advance.
In the case of F25, it is supposed to describe - **after the event took place**, since we are describing past events - what plan the performers executed. In reality, F25 has 2 main practical purposes, one is to link the Performance with all the F22 being performed, and the second is to allow for the description of everything that helps shape a performance but is not part of it in the moment of the performance : for example, a choreographer is not part of the performance, or a metteur en scène, or a video included in the performance, etc., their work happened before the performance and therefore they are part of the "plan" , or the procedure, that led to the performance.
The case of F25 is a special one and it has brought some confusion lately. The first schema used F25 everywhere, on the assumption that F25 was THE way to model the performers' intentions. This was an misinterpretation of FRBRoo's documentation on F25 and furthermore this was sometimes true, sometimes false depending on specific cases, and these cases are impossible to untangle.
So, to sum it up, I assume `foresees creation or performance mode` will be used mostly for F22 and for foreseen events. 
There is still a problem that was raised today regarding the data : do we use référentiels or strings ? It seems that in some cases, for general modes or techniques, a controlled vocabulary would be possible and preferable. In other cases, for specific modes or techniques, a string is the only solution. This corresponds to what CRM describes with P32 and P33, but this is for past events, so we should actually create 2 properties, something like : 
```
foresees general creation or performance mode
foresees specific creation or performance mode
```

**Comment by pierrechoffe** [comment]
So, clearly the situation has a parallel equivalent in the "past events" and we should be able to describe the technique that was used in a performance. For example, if Jimmy Page used a cello bow to play his electric guitar, or if the singers improvised on a tune asked for by the audience.
For most cases I would say that using P32 and P33 would do the job - with the same problem regarding what comes from a controlled vocabulary and what does not.
But I strongly disapprove the present schema that uses F25 to represent these techniques. It is inefficient, costly, inexpressive, misleading and sometimes false.
So basically I would say that for Jimmy Page's performance we have the following:
```
M28 Individual Performance p14_was_carried _out_by E39 Actor Jimmy Page
M28 Individual Performance u1_used_mop M14 MOP electric guitar
M28 Individual Performance p33_used_specific_technique E29 Design or Procedure cello bow
```
instead of p33 we could have `uxxx used specific creation or performance mode`

**Comment by pasqLisena** [comment]
Ok, I need to know precisely WHAT to change and HOW. For now `U90` is used only in itema3 graph

**Comment by pierrechoffe** [comment]
I am not in the group, so I was not at the meetings where they discussed the matter and have no idea who needs this kind of info, so my answer is related to the model in general and based on examples I found in the documentation.
Your question should be addressed to the modelization group through the general email because not everybody follows these comments on github.

**Comment by pasqLisena** [comment]
I am deleting `u86i`: in the [spreadsheet](https://docs.google.com/spreadsheets/d/1RNhHnp4fWEhG8G4acx8FPi-nu9fUWNhc8JNf3SQwUFE/edit#gid=1401336326) `u86` is marked for deletion.
